packages/image-block/src/components/Image.tsx

```diff
@@ -34,7 +34,14 @@ export const ImageComponent = ({
     const link = blockSettings?.hasLink && blockSettings?.linkObject?.link && blockSettings?.linkObject;
     const { isFocused, focusProps } = useFocusRing();
 
-    const Image = <ResponsiveImage image={image} containerWidth={containerWidth} altText={blockSettings.altText} />;
+    const Image = (
+        <ResponsiveImage
+            image={image}
+            containerWidth={containerWidth}
+            altText={blockSettings.altText}
+            style={{ maxWidth: image.width }}
```

Comment:
I don't understand why this was required as a change? It seems like it should have been working before and this behaviour is pretty important for the ResponsiveImage component

Reply:
I figured this part is a bit opinionated, as in the Do & Don't block we don't want this behavior. If the object-fit is set to fill, then we want the image to fill the whole container regardless of the width of the original image.

So I introduced className and style prop to the ResponsiveImage to make it customisable to each use case.